test/src/sdk-harness/test_projects/configurables_in_contract/mod.rs

```diff
@@ -45,7 +47,9 @@ async fn contract_uses_default_configurables() -> Result<()> {
     Ok(())
 }
 
+// TODO Remove ignore when SDK supports encoding V1 for configurables
```

Comment:
please add an issue number for those TODOs

Reply:
Done.